core_gems.rb

```diff
@@ -9,5 +9,5 @@
 download "oj", "3.3.10"
 download "tzinfo", "1.2.10"
 download "tzinfo-data", "1.2016.5"
-fetch "google-protobuf", "3.22.1"
-fetch "grpc", "1.53.0"
+fetch "google-protobuf", "3.25.5"
+fetch "grpc", "1.66.0"
```

Comment:
Is this compatible with `fluent-plugin-google-cloud`? https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/515 used 1.65.2…

Reply:
The presubmits passed with this configuration, so unless the presubmit coverage doesn't go deep enough then I assume it's fine.

Reply:
Ruby allows multiple installed gem versions, so I assume we'll just end up shipping with both 1.66.0 and 1.65.2…